config.go

```diff
@@ -198,3 +216,42 @@ func ConfigLoad() {
 	setupListeners()
 	setupTimeouts()
 }
+
+// IniParser is a parser for config files in classic key/value style format. Each
+// line is tokenized as a single key/value pair. The first "=" delimited
+// token in the line is interpreted as the flag name, and all remaining tokens
+// are interpreted as the value. Any leading hyphens on the flag name are
+// ignored.
+func IniParser(r io.Reader, set func(name, value string) error) error {
+	s := bufio.NewScanner(r)
+	for s.Scan() {
+		line := strings.TrimSpace(s.Text())
+		if line == "" {
+			continue // skip empties
+		}
+
+		if line[0] == '#' {
+			continue // skip comments
+		}
+
+		var (
+			name  string
+			value string
+			index = strings.IndexRune(line, '=')
+		)
+		if index < 0 {
+			name, value = line, "true" // boolean option
+		} else {
+			name, value = strings.TrimSpace(line[:index]), strings.Trim(strings.TrimSpace(line[index+1:]), "\"")
```

Comment:
Yes, quoted strings are still supported for compatibility reasons but only in a simple form (one value, not multiple values). I fixed our examples in the ini file for consistency reasons because we used it in the very beginning. I am not sure if the old parser supported multiple quoted strings per value.

I definitely don't want people to start using quoted strings but it should be compatible for the moment.